stdlib/public/core/Codable.swift

```diff
@@ -2894,13 +2894,28 @@ public extension RawRepresentable where RawValue == String, Self : Decodable {
 //===----------------------------------------------------------------------===//
 
 // FIXME: Uncomment when conditional conformance is available.
-extension Array : Codable /* where Element : Codable */ {
-    public init(from decoder: Decoder) throws {
-        guard Element.self is Decodable.Type else {
-            preconditionFailure("Array<\(Element.self)> does not conform to Decodable because \(Element.self) does not conform to Decodable.")
+extension Array : Encodable /* where Element : Encodable */ {
```

Comment:
@airspeedswift I'm going to merge this in for now; we can revert later or change the implementation if need be.